x11-misc/sunflower/sunflower-0.3.61.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit desktop gnome2-utils python-r1 xdg-utils
+
+MY_PN="Sunflower"
+DESCRIPTION="Small and highly customizable twin-panel file manager with plugin-support"
+HOMEPAGE="https://gitlab.com/MeanEYE/Sunflower
+	http://sunflower-fm.org/"
+SRC_URI="http://sunflower-fm.org/pub/sunflower-0.3-61.tgz"
```

Comment:
Please construct the filename using `${PV}`.

Reply:
Hi @mgorny,
Could you please suggest a proper way to do it?
Should I rename the ebuild to `sunflower-0.3-61.ebuild` or make some other changes?
I didn't use `${PV}` because I didn't find how to do it in a clear way, so I decided that it's better to hardcode the version than to use `sed` to replace last `.` with `-`.
Thanks!